lms/static/sass/course/_student-notes.scss

```diff
@@ -45,6 +45,12 @@ $divider-visual-tertiary: ($baseline/20) solid $gray-l4;
       @extend .content; // needed extend carried over from course handouts UI, but should be cleaned up
       width: 100%;
     }
+
+    .sr-is-focusable,
```

Comment:
I would suggest potentially adding an extend to the _mixins.scss file (common/static/sass/_mixins.scss)  after %cont-text-sr, potentially "%no-focus-sr" so that you can use it in your specific notes application, but also so that other areas could use this styling.